lib/toolbar-button-view.coffee

```diff
@@ -21,9 +21,17 @@ class ToolbarButtonView extends View
       @addClass iconset
 
     @on 'click', =>
+
+      @previouslyFocusedElement = $(document.activeElement);
```

Comment:
Am wondering why you convert the `document.activeElement` to jQuery and later remove it from jQuery again, without using any jQuery API's...

Am also wondering why convert `previouslyFocusedElement` and `eventElement` to members instead of private variables...